dashboard/test/controllers/omniauth_callbacks_controller_test.rb

```diff
@@ -224,4 +224,29 @@ class OmniauthCallbacksControllerTest < ActionController::TestCase
     end
     assert_redirected_to 'http://test.host/home?open=rosterDialog'
   end
+
+  test "omniauth callback sets token on user when passed with credentials" do
+    auth = OmniAuth::AuthHash.new(
+      uid: '1111',
+      provider: 'facebook',
+      info: {
+        name: 'someone',
+        email: 'test@email.com',
+        user_type: User::TYPE_STUDENT,
+        dob: Date.today - 20.years,
+        gender: 'f'
+      },
+      credentials: {
+        token: '123456'
+      }
+    )
```

Comment:
This looks to be rather duplicated across the file. In a follow-up PR, it'd be nice to see this as a constant (`DEFAULT_AUTH_HASH`) or a method (`default_auth_hash`).